draft-age-masque-connect-ip.md

```diff
@@ -350,10 +350,35 @@ forward.
 
 # Examples
 
+CONNECT-IP enables many different use cases that can benefit from IP
+packet proxying and tunnelling. These examples are provided to help
+illustrate some of the ways in which CONNECT-IP can be used.
+
+## Remote Access VPN
+
 The following example shows a point-to-network VPN setup, where a client
 receives a set of local addresses, and can send to any remote server
 through the proxy.
 
+~~~
+
++--------+ IP A         IP B +--------+              +---> IP D
+|        |-------------------|        | IP C         |
+| Client | IP Subnet C <-> * | Server |--------------+---> IP E
+|        |-------------------|        |              |
++--------+                   +--------+              +---> IP ...
+
+~~~
+{: #diagram-tunnel title="VPN Tunnel Setup"}
+
+
+In this case, the client does not specify any scope in its request,
+but is asking to open up a generic IP tunnel. The server assigns
+the client an IPv6 address prefix to client (2001:db8::/64) and
+a full-tunnel route of all IPv6 addresses (::/0). The client
```

Comment:
I like that we have this full-tunnel example, can we also add a split-tunnel "Corporate VPN" example?